### PR TITLE
Made changes in the README for more clarity

### PR DESCRIPTION
**Pull request by Chandak-Keshav**
I added images and commands pertaining to the smooth functioning of the kodyfire. I added the steps to publish the project onto github and how to deal with vulnerabilities which might take place when dealing with npm.
Kindly merge my PR

### CODE REVIEW

**Comment by Chandak-Keshav** [comment]
@anis-marrouchi

File: README.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ npm i react-kodyfire
 kody g react component
 ```
 
+![create your project with your file type](kody1.png)
```
Comment (anis-marrouchi) [inline README.md]: @Chandak-Keshav the explanation is a good idea but it's not part of the cli. `react-kodyfire` is a seperate package with its dedicated readme.

File: README.md [hunk 2]
```diff
@@ -209,6 +216,18 @@ This will create a folder containing the basic structure for a kody. You can sta
 npm i
 ```
 
+To address issues that do not require attention, run:
```
Comment (anis-marrouchi) [inline README.md]: These issues are parts of packages that we need have as seperate package. Thank for reminding me about the issues that require attention. I'll open a new issue to have them fixed. For the now, I believe these should not be part of the desciption

File: README.md [hunk 3]
```diff
@@ -255,6 +274,32 @@ Add the required kodyfire metadata to your package.json
 
 Intialize your project as a git repository and push to a public Github repo
 
+To do so, kindlly follow these steps:-
+1)Intitialsie a new repository in your GitHub and make it as public repository.
```
Comment (anis-marrouchi) [inline README.md]: Let's use a numbered markdown list here instead of manually typing the bullet numbers.

File: README.md [hunk 4]
```diff
@@ -255,6 +274,32 @@ Add the required kodyfire metadata to your package.json
 
 Intialize your project as a git repository and push to a public Github repo
 
+To do so, kindlly follow these steps:-
```
Comment (anis-marrouchi) [inline README.md]: Review the spelling: 
Initialize your project as git repository and push to your public Github repository by following these steps:

1. Initialize a new Github repository and make it public
2. Open your project root folder locally from the terminal and run the following commands: 
```bash
 git init
git add .
git commit -m 'initial commit'
```
3. Link your project to your Github repository
```bash
git remote add origin <copied URL>
git push origin main
```

**Review by anis-marrouchi — CHANGES_REQUESTED** [review]
@Chandak-Keshav please review

File: README.md (hunk 1 above)
Comment (Chandak-Keshav) [reply README.md]: Ok,will work on these right away.

Comment (anis-marrouchi) [reply README.md]: Thank you. I've merged your PR

**Comment by Chandak-Keshav** [comment]
I have made the changes as told.

**Comment by anis-marrouchi** [comment]
Good job @Chandak-Keshav, I've merged your PR

**Pull request merged**